lib/administrate/search.rb

```diff
@@ -30,7 +30,7 @@ def query
     end
 
     def search_terms
-      ["%#{term.downcase}%"] * search_attributes.count
+      ["%#{term.mb_chars.downcase.to_s}%"] * search_attributes.count
```

Comment:
Looking at this and yes, you shouldn't need `#to_s` since you're interpolating and the return value of `String#downcase` is a string.